actions.js

```diff
@@ -276,7 +276,7 @@ export function updateA(self) {
 				},
 			],
 			callback: async (event) => {
-				// ideally, one would not use the aes70 object numbers, but the roles ArrayProcessing/ArrayProcessing_Name1 to 40 to be compatible with other amps
+				// ideally, one would not use the aes70 object numbers, but the roles ArrayProcessing/ArrayProcessing_Enable1 to 40 to be compatible with other amps, where Enable1 is speaker a preset 1, Enable5 speaker a preset 2 etc.
```

Comment:
Is that correct that preset 2 has Path Enable5 ?

Reply:
Yes. It's confusing, but it has the following scheme:
Enable1: Speaker A Preset 1
Enable2: Speaker B Preset 1
Enable3: Speaker C Preset 1
Enable4: Speaker D Preset 1
Enable5: Speaker A Preset 2
etc.

Reply:
@franz-meyer ok the names are so strange because normally every object should have a unique name. But d&b didn't do that so the AES70 libaray had to be changed so that it automatically adds a number to the name.  

I never actually ask, but where do you come from? Should I invite you to the project?

Reply:
I've attached you the rolemapdata.json for the D20:
[rolemapdata.json](https://github.com/AlexK-99/companion-module-d-und-b-amps/files/15468027/rolemapdata.json)

I'm from Germany. Sure, go ahead :)

Reply:
@franz-meyer Ok also auf Deutsch danke für die rolemap und Einladung ist versendet.

Grüße aus dem gerade nicht so sonnigen Coburg(Bayern)
Alexander